.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -3,28 +3,84 @@ name: Bug report
 about: Report a bug for this project
 
 ---
+<!--
+  For community support and other discussions, you are welcome to visit us on our community channels listed at https://mailcow.github.io/mailcow-dockerized-docs/#community-support. For professional commercial support, please check out https://mailcow.github.io/mailcow-dockerized-docs/#commercial-support instead
+-->
 
-**README and remove me**
-For community support and other discussion, you are welcome to visit and stay with us @ Freenode, #mailcow
-Answering can take a few seconds up to many hours, please be patient.
-Commercial support, including a ticket system, can be found @ https://www.servercow.de/mailcow#support - we are also available via Telegram. \o/
+**Prior placing the issue, please check following:** *(fill out each checkbox with a `X` once done)*
+- [ ] I have understood that answers are voluntary and community-driven, and not kind of commercial support.
+- [ ] I do promise I will carefully read below instructions to help getting my issue fixed quickier.
+- [ ] I have verified that my issue has not been already answered in the past. I also checked previous [issues](https://github.com/mailcow/mailcow-dockerized/issues).
+- [ ] I understand that ignoring below instructions might result in immediate closing and deletion of my issue.
 
-**Describe the bug, try to make it reproducible**
-A clear and concise description of what the bug is. How can it be reproduced? 
-If applicable, add screenshots to help explain your problem. Very useful for bugs in mailcow UI.
+---
+
+**Description of the bug**: What kind of issue have you *exactly* come across?
+<!--
+  This should be a clear and concise description of what the bug is. What EXACTLY does happen?
+  If applicable, add screenshots to help explain your problem. Very useful for bugs in mailcow UI.
+  Write your detailed description below.
+-->
+
+My issue is...
+
+**Reproduction of said bug**: How *exactly* do you reproduce the bug?
+<!--
+  Here it is really helpful to know how exactly you are able to reproduce the reported issue.
+  Meaning: What are the exact steps - one by one - to get the above described behavior.
+  Screenshots can be added, if helpful. Add the text below.
+-->
+
+1. I go to...
+2. And then to...
+3. But once I do...
+
+**Expected behavior**: What did you expect to happen instead?
+<!--
+  We now know what kind of issue you are experiencing and how, and the best case, this can be
+  reproduced in a reliable way. Please tell us now, what you expected to happen.
+  This can be quite short usually. Please add text below.
```

Comment:
I would replace `This can be quite short usually.` with `This is usually just a sentence or two.`